cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -1,59 +1,145 @@
-function createNewDashboard(dashboardName) {
-  cy.visit('/dashboards');
-  cy.getByTestId('CreateButton').click();
-  cy.get('li[role="menuitem"]')
-    .contains('Dashboard')
-    .click();
-
+function createNewDashboardByAPI(name) {
   cy.server();
-  cy.route('POST', 'api/dashboards').as('NewDashboard');
-
-  cy.getByTestId('EditDashboardDialog').within(() => {
-    cy.getByTestId('DashboardSaveButton').should('be.disabled');
-    cy.get('input').type(dashboardName);
-    cy.getByTestId('DashboardSaveButton').click();
-  });
-
-  return cy.wait('@NewDashboard').then((xhr) => {
-    const slug = Cypress._.get(xhr, 'response.body.slug');
-    assert.isDefined(slug, 'Dashboard api call returns slug');
+  return cy.request('POST', 'api/dashboards', { name }).then((response) => {
+    const slug = Cypress._.get(response, 'body.slug');
+    assert.isDefined(slug, 'Dashboard api call returns widget slug');
```

Comment:
```suggestion
    assert.isDefined(slug, 'Dashboard api call returns dashboard slug');
```
almost sure the slug is for the dashboard too